spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1348,57 +1371,92 @@
                     ),
                   )
                 end
+              end
 
-                context 'when the enrollment has failed' do
-                  before do
-                    stub_request_failed_proofing_results
-                  end
+              context 'when the enrollment has failed' do
```

Comment:
This was nested incorrectly. I moved it out a level. All specs in this section should just be moved out with the exception of the new spec "updates the profile"